cron/index.js

```diff
@@ -113,6 +113,12 @@ module.exports = {
         module: require('../ee/cron/workspaceHostLoads'),
         intervalSec: config.cronOverrideAllIntervalsSec || config.cronIntervalWorkspaceHostLoadsSec,
       });
+
+      module.exports.jobs.push({
+        name: 'chunksHostLoads',
+        module: require('../ee/cron/chunksHostLoads'),
+        intervalSec: config.cronOverrideAllIntervalsSec || config.cronIntervalChunksHostLoadsSec,
+      });
```

Comment:
Should we wrap this in `isEnterprise()`?

Reply:
Look a few lines up 😉

Reply:
Ah... Serves me right for just looking at the diff in GitHub 😂